layouts/partials/copilot/sidebar.html

```diff
@@ -1,5 +1,6 @@
 {{ $copilotApiUrl := getenv "PULUMI_COPILOT_URL" }}
 {{ if $copilotApiUrl }}
+{{ $copilotBasePath := or (getenv "DEV_PULUMI_COPILOT_BASE_PATH") "/_pulumi/cloud-ai/" }}
 <aside id="ai-sidebar-target"></aside>
-<iframe id="ai-sidebar-host" src="/_pulumi/cloud-ai/sidebar?hostApp=docs&apiHost={{ $copilotApiUrl | urlquery }}"></iframe>
-{{ end }}
+<iframe id="ai-sidebar-host" src="{{ $copilotBasePath }}/sidebar?hostApp=docs&apiHost={{ $copilotApiUrl | urlquery }}"></iframe>
```

Comment:
just noticed this... I think you may end up with a double slash here if the `DEV_PULUMI_COPILOT_BASE_PATH` environment variable is not set, since in that case it would end up with `/_pulumi/cloud-ai//sidebar?hostApp=docs....`